lib/class-sp-sync-meta.php

```diff
@@ -35,7 +35,7 @@ class SP_Sync_Meta {
 	 * @access protected
 	 * @var string
```

Comment:
Should this be a mixed string/boolean based on what the set_transient on line 189 is doing?